src/tools/execution-results.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// Shared execution result checking code
+//
+
+namespace wasm {
+
+static void verifyBitwiseEqual(Literal a, Literal b) {
+  if (a == b) return;
+  // accept equal nans if equal in all bits
+  assert(a.type == b.type);
+  if (a.type == f32) {
+    assert(a.reinterpreti32() == b.reinterpreti32());
+  } else if (a.type == f64) {
+    assert(a.reinterpreti64() == b.reinterpreti64());
+  } else {
+    std::cout << "not bitwise equal\n";
```

Comment:
Would it be useful to log that their types aren't integral, or the values of the literals passed in?
Would that be useful for all the branches of this function?